app/buyers/views/buyers.py

```diff
@@ -246,3 +249,54 @@ def delete_a_brief(framework_slug, lot_slug, brief_id):
             url_for('.view_brief_summary', framework_slug=framework_slug, lot_slug=lot_slug,
                     brief_id=brief_id, delete_requested=True)
         )
+
+
+@buyers.route("/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/answer-question",
```

Comment:
I think it's nice that the URL structure matches the one we have in service submissions. It makes it obvious where to put framework/lot dashboards if we ever need them and it makes the URLs showing up in logs/analytics a lot more useful.

I'm not sure if it's worth moving supplier responses/questions to a similar structure right now though.